test/src/tests-identity.ts

```diff
@@ -2059,159 +2058,6 @@ describe('identity', function() {
         done();
     });
 
-    it("should find the related MPID's cookies when given a UI with fewer IDs when passed to login, logout, and identify, and then log events with updated cookies", async () => {
```

Comment:
This test can remain.  but you can remove the areas associated with products here.  There's still checks for other non-product items that are probably important